clang/lib/Interpreter/Interpreter.cpp

```diff
@@ -248,7 +248,7 @@ Interpreter::~Interpreter() {
 // can't find the precise resource directory in unittests so we have to hard
 // code them.
 const char *const Runtimes = R"(
-    void* operator new(__SIZE_TYPE__, void* __p) noexcept;
+    #include <new>
```

Comment:
```suggestion
```

I'd prefer to drop it for now. We cannot really re-declare the operator new with placement in a platform independent way. We cannot really rely on including `<new>` because it won't work across our build bots.

Reply:
There's a placement `new` just below here (in this block of lines that's always interpreted). I didn't try just removing but I can.

Reply:
Ya so you need something here.